test/test.js

```diff
@@ -458,6 +461,37 @@ function checkRefTestResults(browser, id, results) {
   });
 }
 
+function browserTestReportHandler(req, res) {
+  var parsedUrl = url.parse(req.url, true);
+  var pathname = parsedUrl.pathname;
+  if (pathname === '/browserTestReports' ||
+    pathname === '/browserWorkerTestReports') {
+
+    switch (pathname) {
+      case '/browserTestReports' :
+        const pdfWritableStream = fs.createWriteStream('' +
+          '../coverage/coverageInfo-pdf.json');
+        req.pipe(pdfWritableStream);
+          pdfWritableStream.on('finish', function () {
+          res.end();
+        });
+        break;
+      case '/browserWorkerTestReports' :
+        let workerWritableStream = fs.createWriteStream(
+          '../coverage/coverageInfo-pdfWorker.json');
+        req.pipe(workerWritableStream);
+        workerWritableStream.on('finish', function () {
+          res.end();
+        });
+        req.on('ReceivedCoverageData', function () {
```

Comment:
This event will never fire, so remove it. The type of `req` is `http.IncomingMessage`, and documented at https://nodejs.org/api/http.html#http_class_http_incomingmessage